src/cascadia/TerminalApp/App.cpp

```diff
@@ -342,6 +342,65 @@ namespace winrt::TerminalApp::implementation
         return TermControl::GetProposedDimensions(settings, dpi);
     }
 
+    // Method Description:
+    // - Get the launch mode in json settings file. Now there
+    //   two launch mode: default, maximized. Default means the window
+    //   will launch according to the launch dimensions provided. Maximized
+    //   means the window will launch as a maximized window
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - LaunchMode enum that indicates the launch mode
+    LaunchMode App::GetLaunchMode()
+    {
+        if (!_loadedInitialSettings)
+        {
+            // Load settings if we haven't already
+            LoadSettings();
+        }
+
+        hstring launchModeText = _settings->GlobalSettings().GetLaunchMode();
+        LaunchMode launchMode = LaunchMode::DefaultMode;
+        if (launchModeText == L"maximized")

```

Comment:
I'd like to see the conversion to/from string happening in GlobalAppSettings.cpp, similar to what it does for _requestedTheme.

Reply:
changed